packages/@atjson/renderer-commonmark/src/index.ts

```diff
@@ -233,6 +233,22 @@ function getNumberOfRequiredBackticks(text: string) {
   }, 1);
 }
 
+function isBold(annotation?: any) {
```

Comment:
[Type predicates](https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates) may be _very_ helpful here.

Reply:
Yep, that's what I was looking for. But now I'm not sure if this is the right approach. If we theoretically had access to `document.schema.Bold` then it would be better to rely on that.